frontend/src/components/nodes/ValidatorsList.tsx

```diff
@@ -21,6 +22,24 @@ class ValidatorsList extends Component<Props> {
 
     let validatorsList = validators.sort(
       (a: N.ValidationNodeInfo, b: N.ValidationNodeInfo) => {
+        const aCurrentStake = new BN(a.currentStake)
+          .div(utils.format.NEAR_NOMINATION)
+          .toNumber()
+          .toFixed(0);
+        const bCurrentStake = new BN(b.currentStake)
+          .div(utils.format.NEAR_NOMINATION)
+          .toNumber()
+          .toFixed(0);
+
+        // sort validators by 'proposedStake' if 'currentStake' of those validators are same
+        // that's why we need to round 'currentStake' to 'toFixed(0)'
+        if (
+          (a.proposedStake || b.proposedStake) &&
+          new BN(bCurrentStake).eq(new BN(aCurrentStake))
+        ) {
+          return new BN(b.proposedStake || 0).sub(new BN(a.proposedStake || 0));
+        }
+
```

Comment:
1. Yeah, correct. There is no need to convert to BN

2. I've added that condition because we compare not actual validators value but rounded. That's why when I remove condition of proposedStake the validators ordering will be wrong:
![Снимок экрана 2021-09-07 в 10 15 01](https://user-images.githubusercontent.com/34593263/132302178-d5a30ee9-3870-43c1-ad5e-df2a9771714b.jpg)
So let's leave it as is.